pkgs/development/python-modules/latexrestricted/default.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  fetchPypi,
+  buildPythonPackage,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "latexrestricted";
+  version = "0.4.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-AMfDTruJKejDdXW98VkeeEwELql5566bsL1SutLDpso=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
```

```suggestion
  pythonImportsCheck = [ "latexrestricted" ];
  
  # upstream has no tests
  doCheck = false;
```
Could you add `pythonImportsCheck` for a smoke check?
Also, we encourage enabling tests, so we'd like to explicitly disable `checkPhase` if there is no tests.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages

---

Done.